pkg/api/api.go

```diff
@@ -70,6 +70,7 @@ func (hs *HTTPServer) registerRoutes() {
 	r.Get("/admin/orgs", authorizeInOrg(reqGrafanaAdmin, acmiddleware.UseGlobalOrg, orgsAccessEvaluator), hs.Index)
 	r.Get("/admin/orgs/edit/:id", authorizeInOrg(reqGrafanaAdmin, acmiddleware.UseGlobalOrg, orgsAccessEvaluator), hs.Index)
 	r.Get("/admin/stats", authorize(reqGrafanaAdmin, ac.EvalPermission(ac.ActionServerStatsRead)), hs.Index)
+	r.Get("/admin/usage-report-preview", authorize(reqGrafanaAdmin, ac.EvalPermission(ac.ActionServerStatsRead)), hs.Index)
```

Comment:
Why are we registering this route, there don't seem to be any frontend associated?
![image](https://user-images.githubusercontent.com/5232696/149089691-859e7fdc-e65c-4c33-b59e-38aea185922e.png)

Reply:
Indeed, we should not register it.

Context: Talked with Mitch about frontend, he thinks for now we can just expose an API endpoint